src/modules/API.js

```diff
@@ -83,7 +76,6 @@ export const getComment = async (id) => {
     const data = await response.json();
     return data;
   } catch (error) {
-    console.error(error);
+    throw new Error('Something went wrong!');
```

Comment:
Please consider changing all of them 
Example : `throw new Error('Something went wrong!', error );`